src/api/stock.ts

```diff
@@ -12,7 +12,7 @@ export default ({ config, db }) => {
 
   const _getStockId = (storeCode) => {
     let storeView = config.storeViews[storeCode]
-    return storeView ? storeView.msi.stockId : config.msi.defaultStockId
+    return storeView  && storeView.msi && storeView.msi.stockId ? storeView.msi.stockId : config.msi.defaultStockId
```

Comment:
As we have typescript version 3.7 available we can use [Optional Chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) which in my opinion is much cleaner when the "&&" chain. 

```suggestion
    return storeView?.msi?.stockId ? storeView.msi.stockId : config.msi.defaultStockId
```